README.md

```diff
@@ -1,28 +1,36 @@
 # cluster-api-provider-juju
-// TODO(user): Add simple overview of use/purpose
+
+A Cluster API infrastructure provider which uses [Juju][].
 
 ## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
+
+This Cluster API infrastructure provider manages a Juju model containing [Charmed Kubernetes][] components.
+This provider is used alongside the [Charmed Kubernetes Control Plane Provider][] and the [Charmed Kubernetes Bootstrap Provider][]. These providers support deploying Charmed Kubernetes 1.27+. It does not work with other Cluster API providers.
 
 ## Getting Started
 
 ### Creating the management cluster
-You’ll need a Kubernetes cluster to run against. Assuming you have a juju controller, cloud, and cloud credentials set up already, you can add a new model and deploy Kubernetes-Core. Note: Your cluster will need storage capabilties, so ensure your overlay contains the cloud provider/integrator required to set that up. 
+
+You’ll need a Kubernetes cluster to host the various controllers and resources. Charmed Kubernetes can be used for this purpose if you wish. Assuming you have a Juju controller, cloud, and cloud credentials set up already, you can add a new model and deploy Kubernetes-Core. Note: Your cluster will need storage and load-balancing capabilties, so ensure your overlay contains the cloud provider/integrator required to set that up. For more details on installing Charmed Kubernetes, see the [Charmed Kubernetes documentation][]
```

Comment:
Looks like this `[Charmed Kubernetes documentation][]` bit is missing a link. It's coming through unrendered in the [preview](https://github.com/charmed-kubernetes/cluster-api-provider-juju/blob/6f02824aadbd235b88bc0ff69eef814d5bad4be0/README.md).

Reply:
my fault, will fix